src/Core/src/Graphics/Polygon.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Graphics;
+
+namespace Microsoft.Maui.Graphics
+{
+	public class Polygon : Shape
+	{
+		public Polygon()
+		{
+
+		}
+
+		public Polygon(PointCollection? points)
+		{
+			Points = points;
+		}
+
+		public PointCollection? Points { get; set; }
+
```

Comment:
Missing for adding a property of type `FillRule`:
- EvenOdd
- Nonzero

@Clancey @jonlipsky Could we add in Microsoft.Maui.Graphics a property/method to set how the intersecting areas contained in the Polygon(Path) are combined?

Reply:
I just saw this comment.  If you didn't see it, it is supported in Maui.Graphics:

https://github.com/dotnet/Microsoft.Maui.Graphics/blob/a46ffe17dae9cd9ec339eceac524541b0d801c24/src/Microsoft.Maui.Graphics/ICanvas.cs#L26

WindingMode